templates/etc/init.d/deb/td-agent

```diff
@@ -81,11 +81,8 @@ if [ -n "${TD_AGENT_PID_FILE}" ]; then
   TD_AGENT_ARGS="${TD_AGENT_ARGS} --daemon ${TD_AGENT_PID_FILE}"
 fi
 
-# 2012/04/17 Kazuki Ohta <k@treasure-data.com>
-# Use jemalloc to avoid memory fragmentation
-if [ -f "${TD_AGENT_HOME}/embedded/lib/libjemalloc.so" ]; then
-  export LD_PRELOAD="${TD_AGENT_HOME}/embedded/lib/libjemalloc.so"
```

Comment:
Should we also remove the build of jemalloc? Or do we want to package it in case customers want to switch back?

Reply:
I'd rather not modify the build, unless required. On one hand, it could be useful. On the other, as you were saying offline, the current build system is hard to reason about.

Reply:
This was a question rather than a request, so we can definitely defer it.